src/avram/factory.cr

```diff
@@ -1,11 +1,17 @@
 abstract class Avram::Box
+  macro inherited
+    {% raise "Avram::Box has been renamed to Avram::Factory" %}
+  end
+end
+
+abstract class Avram::Factory
   getter operation
 
   SEQUENCES = {} of String => Int32
 
   macro inherited
     {% unless @type.abstract? %}
-      {% operation = @type.name.gsub(/Box/, "::SaveOperation").id %}
+      {% operation = @type.name.gsub(/Factory$/, "::SaveOperation").id %}
```

Comment:
Added the `$` here to ensure it's replacing the ending `Factory` word in case someone has a model called something like `FactorySetting`